scripts/sonic_sku_create.py

```diff
@@ -56,7 +56,7 @@
     "1x1":   { "lanes":4, "speed":1000,   "step":4, "bko":0, "name": "etp" },
     "4x10":  { "lanes":4, "speed":10000,  "step":1, "bko":1, "name": "etp" },
     "4x25":  { "lanes":4, "speed":25000,  "step":1, "bko":1, "name": "etp" },
-    "2x50":  { "lanes":4, "speed":25000,  "step":2, "bko":1, "name": "etp" },
+    "2x50":  { "lanes":4, "speed":50000,  "step":2, "bko":1, "name": "etp" },
```

Comment:
>50000 [](start = 34, length = 5)

You mentioned "When trying to use port split feature of SKU creator tool to split a port in port_config.ini and config_db.json. It was not working".
I am wondering if this bug is not caught in unit test, and if there is a gap in unit test?

Reply:
The unit test in sonic-utilities for sonic_sku_create tool does not cover testing individual sub features. This is some how missed in manual testing and later when I tried to test it for documentation purposes, I found this issue. And, so addressed it.

Reply:
I see. Could you add more unit testcases to cover the changes here?

---
In reply to: [568833823](https://github.com/Azure/sonic-utilities/pull/1380#discussion_r568833823) [](ancestors = 568833823)

Reply:
Adding more unit test cases adds more changes to the sonic_sku_create.py script itself, that is why we had a basic unit test case to cover the main feature of the tool was written. But,  the tool itself is well tested manually now.